index.js

```diff
@@ -961,7 +961,7 @@ export var detectedKeywords = null;
  * This also replicates JQuery `$(document).ready()`, with added protection for
  * usage of `async` -- the DOM ready event can fire before the script is loaded..
  */
-if (require.main !== module && check_dependencies()) {
+if (check_dependencies()) {
```

Comment:
We talked about this one a bit when talking about how to surface this library in addons. `require.main` works for only a very specific check, and doesn't actually help the use case of determining whether this library is executed as a module or as a browser script -- there is a method for this being included in one of the next ES, but that is a ways off.

This was just required to get this working as `require` doesn't exist when treating the library as an ES6 module.